sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/_tests_/e2e/application-offering-change-request.institutions.controller.getCompletedApplications.e2e-spec.ts

```diff
@@ -0,0 +1,320 @@
+        {
+          institutionLocation: collegeFLocation,
+        },
+        {
+          initialValues: {
+            applicationOfferingChangeRequestStatus:
+              ApplicationOfferingChangeRequestStatus.Approved,
+          },
+        },
+      );
+    // Student 4 has an in progress with student application offering change request with a different institution.
+    await saveFakeApplicationOfferingRequestChange(db);
+    // Student 5 has a declined by student application application offering change request.
+    const declinedByStudentApplicationOfferingChange =
+      await saveFakeApplicationOfferingRequestChange(
+        db,
+        {
+          institutionLocation: collegeFLocation,
+        },
+        {
+          initialValues: {
+            applicationOfferingChangeRequestStatus:
+              ApplicationOfferingChangeRequestStatus.DeclinedByStudent,
+          },
+        },
+      );
+    // Student 6 has a declined by SABC application offering change request.
+    const declinedBySABCApplicationOfferingChange =
+      await saveFakeApplicationOfferingRequestChange(
+        db,
+        {
+          institutionLocation: collegeFLocation,
+        },
+        {
+          initialValues: {
+            applicationOfferingChangeRequestStatus:
+              ApplicationOfferingChangeRequestStatus.DeclinedBySABC,
+          },
+        },
+      );
+
+    approvedApplicationOfferingChange.assessedDate = new Date();
+    declinedByStudentApplicationOfferingChange.studentActionDate = new Date();
+    declinedBySABCApplicationOfferingChange.assessedDate = new Date();
+
+    await db.applicationOfferingChangeRequest.save([
+      approvedApplicationOfferingChange,
+      declinedByStudentApplicationOfferingChange,
+      declinedBySABCApplicationOfferingChange,
+    ]);
+
+    const applicationWithApprovedApplicationOfferingChange =
+      approvedApplicationOfferingChange.application;
+    const applicationWithDeclinedByStudentApplicationOfferingChange =
+      declinedByStudentApplicationOfferingChange.application;
+    const applicationWithDeclinedBySABCApplicationOfferingChange =
+      declinedBySABCApplicationOfferingChange.application;
+
+    applicationWithApprovedApplicationOfferingChange.applicationNumber =
+      "1000000002";
+    applicationWithDeclinedByStudentApplicationOfferingChange.applicationNumber =
+      "1000000001";
+    applicationWithDeclinedBySABCApplicationOfferingChange.applicationNumber =
+      "1000000000";
+
+    await db.application.save([
+      applicationWithApprovedApplicationOfferingChange,
+      applicationWithDeclinedByStudentApplicationOfferingChange,
+      applicationWithDeclinedBySABCApplicationOfferingChange,
+    ]);
+
+    const endpoint = `/institutions/location/${collegeFLocation.id}/application-offering-change-request/completed?page=0&pageLimit=10&sortField=applicationNumber&sortOrder=${FieldSortOrder.ASC}`;
+    const institutionUserToken = await getInstitutionToken(
```

Comment:
May be we can have the institutionUserToken assigned to a global variable at `beforeAll` and used across the tests.

```ts
let institutionFUserToken ;
beforeAll(){
institutionFUserToken = await getInstitutionToken(
      InstitutionTokenTypes.CollegeFUser,
    );
}
```